### PR TITLE
Newsweek-Using Bootstrap

### PR DESCRIPTION
**Pull request by Cathella**
This project required that we replicate the Newsweek Homepage using Bootstrap a CSS grid framework. I used boostrap to create this project in its responsive nature. I also exploited some of the utility classes just to make all elements appear well on the homepage throughout all the screen viewports.

I considered creating the desktop view first then continued to make it responsive with smaller and bigger screen sizes using the .col-md-*, .col-sm-* and .col-lg-* bootstrap classes.
Since it is a framework, these classes are already defined in the boostrap.css minified file.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
### Status: Changes required
Amazing work @Cathella  Your site looks amazing and almost similar to the original page. We have to fix a couple of small issues before moving into the next project. Please make the following changes and submit the project for another review. :point_down: 

[Required] Please make your second navbar similar to that of original one by adding **search icon, red box at the end, and give some margin between navlinks.**

IN YOUR PROJECT
Screenshot from 2020-04-03 10-01-40

**But it should be like this :point_down:  :point_down:** 
Screenshot from 2020-04-03 10-01-55

**Comment by sumancrest0001** [comment]
[REQUIRED] Please reduce some margin to make it similar to that of the original one.
Screenshot from 2020-04-03 10-00-37
Screenshot from 2020-04-03 10-00-05

But we have to make it like this :point_down: :point_down: 
Screenshot from 2020-04-03 10-00-22
Screenshot from 2020-04-03 10-00-50

**Comment by sumancrest0001** [comment]
[Required] We have to fix the layout of on medium screen. Please have a look.
Screenshot from 2020-04-03 10-05-38

Keep the "© 2020 NEWSWEEK" at the baseline.

**Comment by sumancrest0001** [comment]
[REQUIRED] we have to fix these issues on the small screen.

in your project

Screenshot from 2020-04-03 10-04-16


**Contents of the subscribe and newsletter should be at the center :point_down:** 

Screenshot from 2020-04-03 10-04-51

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1147 @@
...
+                    <img class="img-fluid" src="https://d.newsweek.com/en/full/1563375/delphian-school.jpg?w=500&h=333&f=c4fe893bfb5642271e1298b1bea3989b" alt="Image">
+                    <a class="image-badge sponsored" href="#">SPONSORED INSIGHT</a>
+                    <div class="need-padding">
+                        <h3>
+                            <a href="#">What Can A Private Education Do For Your Child?</a>
+                        </h3>
+                        <p>What is the single most consequential choice you make to give your child the best possible advantage in life? If you think it’s education, read on.</p>
+                    </div>
+                </div> -->
+                <!-- <div id="sponsored-insight" class="border">
```
Comment (sumancrest0001) [inline index.html]: [REQUIRED] Please remove all the commented codes. If you want to go back, use git command.

**Comment by sumancrest0001** [comment]
**All the best with the next submission. :+1:  :+1: 
Need help? React out to me on slack  suman shrestha**

**Comment by Dandush03** [comment]
### :heavy_check_mark: Project Approve :boom:  :checkered_flag:
# Hey @Cathella  :hand: 
### Great Job so Far :grey_exclamation: :grey_exclamation: :muscle: :smiley: 

### Following Previous Reviews and Seeing you Have Made Required Changes. 
### The project Looks Great.:grey_exclamation::wink:

### Good Luck in your Next Assigment
### Happy coding:computer: 
#### Daniel Laloush

**Pull request merged**